src/fides/api/api/v1/urn_registry.py

```diff
@@ -152,14 +152,16 @@
 ACCESS_MANUAL_WEBHOOK = CONNECTION_BY_KEY + "/access_manual_webhook"
 
 # Collection URLs
+DATASETS = "/dataset"
+DATASET_CONFIG = "/datasetconfig"
 DATASET_VALIDATE = CONNECTION_BY_KEY + "/validate_dataset"
-DATASETS = CONNECTION_BY_KEY + "/dataset"
-DATASET_CONFIGS = CONNECTION_BY_KEY + "/datasetconfig"
-DATASET_BY_KEY = CONNECTION_BY_KEY + "/dataset/{fides_key}"
-DATASETCONFIG_BY_KEY = CONNECTION_BY_KEY + "/datasetconfig/{fides_key}"
+CONNECTION_DATASETS = CONNECTION_BY_KEY + DATASETS
```

Comment:
I refactored this into `CONNECTION_DATASETS` because I needed `DATASETS` for the new route